jest.config.js

```diff
@@ -7,6 +7,13 @@ module.exports = {
   // Automatically clear mock calls and instances between every test
   clearMocks: true,
 
+  globals: {
+    'ts-jest': {
+      // Workaround for https://github.com/kulshekhar/ts-jest/issues/1648.
+      tsconfig: 'tsconfig.base.json',
```

Comment:
Without this, config options from the tsconfig.base aren't getting picked up properly during tests (see https://github.com/kulshekhar/ts-jest/issues/1648). This became apparent when adding tests using React, since the `jsx` option wasn't getting set, causing the tests to not work.